app/controllers/comments_controller.rb

```diff
@@ -1,11 +1,25 @@
 class CommentsController < ApplicationController
-  def index
-    @comments = Comment.all
-    render status: :ok
+  def new
+    @comment = Comment.new
   end
 
-  def show
-    @comment = Comment.find(params[:id])
-    render status: :ok
+  def create
+    @comment = Comment.new(comment_params)
+    @post = Post.find(params[:post_id])
+    @comment.author = User.find(params[:user_id])
```

Comment:
- [ ] The author of the comment should be the `current_user`, not user from the URL route

Reply:
- [ ] @AlyAtalla Great work implementing the changes requested, however you have not implemented the change requested, kindly ensure the author of the comment is the current_user and not the user from the routes.